packages/ui-extensions/docs/surfaces/admin/create-doc-files.js

```diff
@@ -97,7 +97,7 @@ import shared from './shared';
 
 const data: ReferenceEntityTemplateSchema = {
   ...shared,
-  category: 'Components',
+  category: 'Polaris web components',
```

Comment:
This is just a template so it won't update any docs. Looks like the right change though to ensure incorrect category doesn't continue to get added.